pkg/apis/configuration/validation/validation_test.go

```diff
@@ -946,7 +946,7 @@ func TestValidateRedirectURL(t *testing.T) {
 			msg:         "x-forwarded-proto redirect url use case",
 		},
 		{
-			redirectURL: "${host}${request_uri}",
+			redirectURL: "http://${host}${request_uri}",
```

Comment:
should this be `${scheme}`?

Reply:
As it says in the `msg`, this case was specific for: `"use multi variables, no scheme set",` but I am happy to add another use case with all variables if it makes sense to you.